### PR TITLE
Remove GOVUK Elements [Part 8]

### PR DESCRIPTION
**Pull request by tombye**
This is part 8 of a series of work to remove the GOVUK Elements frontend library from our codebase:

https://www.pivotaltracker.com/story/show/182596680

The intention of these changes was originally to remove all GOVUK Elements CSS from the page we would use if the app ever needed to be taken offline for maintenance (see https://github.com/alphagov/notifications-admin/pull/3164 for more details). Because this page only contains a few bits of basic content I decided to just redo its CSS to only include styles from GOVUK Frontend, the library for the GOV.UK Design System. Full details are in the commits.

### CODE REVIEW

File: paas-failwhale/index.html [hunk 1]
```diff
@@ -16,8 +16,7 @@
     <link rel="apple-touch-icon" sizes="152x152" href="/static_503/images/govuk-apple-touch-icon-152x152.png?40846d46aa37232e2b35065769ce567c">
     <link rel="apple-touch-icon" href="/static_503/images/govuk-apple-touch-icon.png?3c45daa1f1e716458e4da954b973002e">
 
-    <link rel="stylesheet" media="screen" href="/static_503/stylesheets/main.css?d077f86473501ab244de0b30600536ee" />
-    <link rel="stylesheet" media="print" href="/static_503/stylesheets/print.css?28010888ca5719dc83d7c2c80f6ed2b2" />
+    <link rel="stylesheet" media="all" href="/static_503/stylesheets/main.css?d077f86473501ab244de0b30600536ee" />
```
Comment (klssmith) [inline paas-failwhale/index.html:19]: If we're using this query string for cache clearing purposes (I'm guessing?) do we need to change it now that the file has changed?

**Review by klssmith — COMMENTED** [review]
Looks like we do use print.css [here](https://github.com/alphagov/notifications-admin/blob/d25cf3030296a93569d77c0e8b3403988683098c/app/assets/error_pages/5xx.html#L20) as well as for the failwhale - not sure if that means we do still need it after all or if it's still fine to get rid of and to remove all references to. Although [this is our main 500 page](https://github.com/alphagov/notifications-admin/blob/d25cf3030296a93569d77c0e8b3403988683098c/app/templates/error/500.html) 🤔

**Comment by tombye** [comment]
> Looks like we do use print.css [here](https://github.com/alphagov/notifications-admin/blob/d25cf3030296a93569d77c0e8b3403988683098c/app/assets/error_pages/5xx.html#L20) as well as for the failwhale - not sure if that means we do still need it after all or if it's still fine to get rid of and to remove all references to. Although [this is our main 500 page](https://github.com/alphagov/notifications-admin/blob/d25cf3030296a93569d77c0e8b3403988683098c/app/templates/error/500.html) 🤔

I'm pretty sure those error pages are served by the same server which serves our static assets and use the same stylesheets as our app. Our app still uses separate stylesheets for screen and print styles so we still need `print.css`. The failwhale is a static app, served by PaaS when our apps and servers aren't online. Because of that, it needs all its CSS packaged up with it, separately to our app pages.

File: paas-failwhale/index.html (hunk 1 above)
Comment (tombye) [reply paas-failwhale/index.html]: We do! Should now be added in https://github.com/alphagov/notifications-admin/pull/4330/commits/d6467daa1d1249cfccf3af8805c73bddb31bb81f, with details of how I got the hash.

**Review by klssmith — APPROVED** [review]

**Pull request merged**